detection-rules/attachment_sus_employee_doc.yml

```
@@ -8,7 +8,7 @@ source: |
     // the subject contains pay related items
     (
       strings.icontains(subject.subject, 'salary')
-      or regex.icontains(subject.subject, '\bpay(?:roll|\b)')
+      or regex.icontains(subject.subject, '\bpay(out|roll|\b)')
```

keep this as a non-capturing group (nitpick)
```suggestion
      or regex.icontains(subject.subject, '\bpay(?:out|roll|\b)')
```